deploy-board/deploy_board/templates/configs/env_config.tmpl

```diff
@@ -91,19 +91,17 @@
                                type="text" value="{{ env.chatroom |default_if_none:'' }}"
                                maxlength="128"/>
                     </div>
-                    <label for="notify" class="deployToolTip control-label col-xs-2"
+
+                    <label for="mentionRecipients" class="deployToolTip control-label col-xs-2"
                         data-toggle="tooltip"
-                        title="Whether or not to notify commiters when deploy">
-                        Notify Authors
+                        title="Ping slack user or slack user group in the chat rooms (slack channels), such as user1,userGroup2">
+                        Ping User or User Group
                     </label>
 
                     <div class="col-xs-4">
-                    {% if env.notifyAuthors %}
-                        <input class="" id="notify_author" name="notify_author" type="checkbox" value="" checked>
-                    {% else %}
-                        <input class="" id="notify_author" name="notify_author" type="checkbox" value="">
-                    {% endif %}
-                    </div>
+                        <input class="form-control" name="mention_recipients" required="false"
+                               type="text" value="{{ env.mentionRecipients |default_if_none:'' }}"/>
+                    </div>                    
```

Comment:
Not a blocker. It would be nice to have some front-end validations for Ping User Or User Group input. A `placeholder` with example for the input text box may be easier to see (besides `title`).

Reply:
Make sense. The same improvement can apply to all other fields.